news/fix_is_current.rst

```diff
@@ -1,5 +1,5 @@
 **Added:**
-    * New date functions to dates.py
+    * New date functions to dates.py that utilize Python's datetime module
```

Comment:
I think what is needed is to list the new functionality, so is_current, etc.  Though the functions already existed but actually you have factored them to use datetime objects, in which case it shouldn't be in "added" but in "changed"

Reply:
So should I delete what I mentioned in the `removed` section?